app/assets/javascripts/admin/adapters/web-hook-event.js.es6

```diff
@@ -0,0 +1,7 @@
+import RESTAdapter from 'discourse/adapters/rest';
+
+export default RESTAdapter.extend({
+  basePath() {
+    return '/admin/api/';
```

Comment:
Just wondering: does this work when using subfolder? (Might have to replace this with `Discourse.getUrl("/admin/api/")`)